src/components/Rating/Stars.tsx

```diff
@@ -0,0 +1,13 @@
+import { Star } from 'lucide-react'
+
+export default function Stars({ rating }: { rating: number }) {
+  const renderStars = () => {
+    const stars = []
+    for (let i = 0; i < 5; i++) {
+      const starColor = i < rating ? '#FFD700' : '#CCCCCC'
```

Comment:
[SUG] We should avoid hardcoding colors

```suggestion
import colors from 'tailwindcss/colors'
import { Star } from 'lucide-react'

export default function Stars({ rating }: { rating: number }) {
  const renderStars = () => {
    const stars = []
    for (let i = 0; i < 5; i++) {
      const starColor = i < rating ? colors.yellow[500] : colors.gray[300]
```